Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node
+
+WORKDIR /app
+
+COPY scripts/wait-for.sh scripts/wait.sh scripts/query.js ./
+RUN chmod +x ./wait-for.sh
+
+RUN npm install mongodb
+RUN npm install argon2 nanoid
```

Comment:
you need to run `yarn install --frozen-lockfile` here as this container runs the whole application.
and in the end run entrypoint.sh

Reply:
and after that `yarn run build --prod`